include/jemalloc/internal/jemalloc_internal.h.in

```diff
@@ -145,6 +145,13 @@ static const bool have_thp =
     false
 #endif
     ;
+static const bool have_getcpu =
```

Comment:
This feature can't currently be supported on all platforms, so the main idea would be to provide introspection capability re: whether the feature is available.  However, as I look a bit closer at other similar situations, I think ```have_dss``` is actually what we should mimic, so this would be controlled via ```have_percpu_arena``` and ```JEMALLOC_PERCPU_ARENA```.  So, with that in mind, what I'd like to see is a separation between these and ```JEMALLOC_HAVE_SCHED_GETCPU```, which is just one possible implementation mechanism.